integration/images/volume-ownership/Dockerfile

```diff
@@ -12,6 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# TODO: use ghcr.io
 FROM ubuntu
```

Comment:
This is tricky. @dims has switched from Busybox to Ubuntu in #5416 due to 

> busybox now does not have nogroup, since the test needs it, switch over from busybox to ubuntu for just the volume-ownership image

But having three distros (Alpine, Busybox and Ubuntu) seems excessive for me. Having nogroup in either Busybox or Alpine is totally possible I suppose.

Reply:
@dims Are we building this image on GitHub Actions? If we manually build this image occasionally, we probably don't have to worry about that right now.

Reply:
I'm gonna use Alpine for now.

Reply:
@kzys yes we build auto-magically, but using `prow` ( https://github.com/kubernetes/test-infra/blob/b62afa2c3b1ca8baafe8488b9371a36f805a634c/config/jobs/kubernetes/sig-node/containerd.yaml#L86-L108 )